SQLCache.js

```diff
@@ -1,51 +1,33 @@
 const { InMemoryLRUCache } = require("apollo-server-caching");
-const DataLoader = require("dataloader");
+const crc = require("crc");
 
 class SQLCache {
-  constructor(cache = new InMemoryLRUCache(), knex) {
+  constructor(cache = new InMemoryLRUCache(), knex, options = {}) {
     this.cache = cache;
-    this.loader = new DataLoader(rawQueries =>
-      Promise.all(rawQueries.map(rawQuery => knex.raw(rawQuery)))
-    );
+    this.hashing = options.hashing;
   }
 
-  getCacheKeyForQuery(query) {
+  getCacheKeyForQuery(query, ttl) {
     const queryString = query.toString();
+    if (this.hashing) {
+      return `sqlcache:${crc.crc32(queryString)}_${ttl}`;
```

Comment:
does the TTL need to be in the key?

Reply:
yes, it`s help controll dinamic cahs time